tests/test_validate.py

```diff
@@ -766,6 +768,44 @@ def test_nested_html_directory_violations(
                 )
 
 
+@pytest.fixture
+def test_data_path():
+    return Path(__file__).parent / "data/validate_mbz"
+
+
+@pytest.fixture
+def question_xml(test_data_path):
+    with open(test_data_path / "questions.xml", "r") as f:
+        questions = f.read()
+    return questions
```

Comment:
Minor: It's nicer to have all fixtures at the top of the file just so it's easier to spot them for re-use in future test cases.